### ISSUE TITLE
Bug in "Sort by name" and "Select certain fields" in demo

### ISSUE THREAD

**Issue by kenfehling** [nolanlawson/pouchdb-find#136]
In the [demo](http://nolanlawson.github.io/pouchdb-find/) the use of ```$exists``` causes an error in the "Sort by name" and "Select certain fields" examples.
```
Error: couldn't find a usable index. try creating an index on: name.
Make sure that only $eq, $gt, $gte, $lt, and $lte are used for the indexed fields.
```

**Comment by nolanlawson** [comment]
Yeah, this was introduced recently because we changed the semantics of `$exists` to exactly match those of Cloudant. It should be chnaged to `$gt: null`.

**Comment by nolanlawson** [comment]
fixed by 67701dc83411db1e57a7366228e330de41b14a29
